.drone.yml

```diff
@@ -24,8 +24,7 @@ steps:
   pull: always
   settings:
     image_name: java8-mvn:${DRONE_COMMIT_SHA}
-    # Whitelisting CVE-2021-26291 temporarily as the recommended Maven version still contains shared-utils from vulnerable v3.2.1
-    whitelist: CVE-2019-5827, CVE-2020-14363, CVE-2021-25215, CVE-2021-27219, CVE-2021-26291
+    whitelist: CVE-2019-5827, CVE-2020-14363, CVE-2021-25215, CVE-2021-27219, CVE-2021-31535, CVE-2019-5827, CVE-2021-37714
```

Comment:
I believe you should be able to get rid of CVE-2020-14363, CVE-2021-25215, CVE-2021-27219 as it they aren't in the list of vulnerabilities you linked me to (https://drone-gh.acp.homeoffice.gov.uk/UKHomeOffice/docker-java8-mvn/70/1/5)

Reply:
@DJamesHO Hi. This has been now fixed. Thanks.